core/src/main/java/bio/terra/pearl/core/service/kit/PepperDSMClientProvider.java

```diff
@@ -0,0 +1,35 @@
+package bio.terra.pearl.core.service.kit;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+
+@Configuration
+public class PepperDSMClientProvider {
```

Comment:
This is a little bit of an experiment based on prior experiences with JavaEE/CDI. This might be able to be accomplished with more annotations and less code, but for now it's doing what I want. Feedback welcome.